xbmc/interfaces/json-rpc/JSONServiceDescription.cpp

```diff
@@ -768,7 +768,7 @@ JSONRPC_STATUS JSONSchemaTypeDefinition::Check(const CVariant &value, CVariant &
       // allowed there is no need to check every element
       if (value.size() < items.size() || (value.size() != items.size() && additionalItems.size() == 0))
       {
-        CLog::Log(LOGDEBUG, "JSONRPC: One of the array elements does not match in type {0}", name.c_str());
```

Comment:
Mhm, looks like it was ported to use StringUtils::Format internally (unless I oversee something):
https://github.com/xbmc/xbmc/blame/master/xbmc/utils/log.cpp